config/jobs-chromeos.yaml

```diff
@@ -1,5 +1,43 @@
 _anchors:
 
+  kbuild-clang-17-arm64-chromeos: &kbuild-clang-17-arm64-chromeos-job
+    template: kbuild.jinja2
+    kind: kbuild
+    image: ghcr.io/kernelci/{image_prefix}clang-17:arm64-kselftest-kernelci
+    params: &kbuild-clang-17-arm64-chromeos-params
+      arch: arm64
+      compiler: clang-17
+      cross_compile: 'aarch64-linux-gnu-'
+      defconfig: 'cros://chromeos-{krev}/{crosarch}/chromiumos-{flavour}.flavour.config'
+      flavour: '{crosarch}-generic'
+      fragments:
+        - lab-setup
+        - arm64-chromebook
+        - CONFIG_MODULE_COMPRESS=n
+        - CONFIG_MODULE_COMPRESS_NONE=y
+      kselftest: disable
```

Comment:
Those params (except `arch`, `cross_compile` and `fragments`) could be factored separately and re-used in the job definitions, adding only the arch-specific ones.